lib/active_yaml/base.rb

```diff
@@ -18,9 +18,15 @@ def extension
       end
 
       private
+if Psych::VERSION >= "4.0.0"
+      def load_path(path)
+        YAML.load(ERB.new(File.read(path)).result, aliases: true, permitted_classes: [Time, Date, Symbol])
```

Comment:
We may need to consider using `unsafe_load` for backwards compatibility.
`YAML.unsafe_load` in Psych 4.0 is same as `YAML.load` in Psych < 4.0.

Reply:
ok, so just use `unsafe_load` and remove the need for the `permitted_classes` and gook